google/cloud/datastore/aggregation.py

```diff
@@ -143,14 +146,17 @@ def _to_pb(self):
             pb.aggregations.append(aggregation_pb)
         return pb
 
-    def count(self, alias=None):
+    def count(self, alias=None, limit=None):
```

Comment:
`limit` is passed in on the `fetch` call for existing queries. It may make more sense to stay consistent and move the limit parameter into fetch instead. What do you think?